rdrf/rdrf/lookup_views.py

```diff
@@ -185,6 +80,11 @@ def _get_relationships(self):
         return [pair[0] for pair in PatientRelative.RELATIVE_TYPES]
 
 
+# TODO I think that for this one the get will be replaced by Django REST framework view
+# The put that switches contexts should be moved. It isn't a lookup view for sure, but also
+# it doesn't feel "right" to mix it into the REST API because it changes things in the
+# session of a user that is using the web UI not consuming the API
+
```

Comment:
Agree, kind of. Session context_id set/get should be separate from the api for getting contexts. Whether it's within the DRF api or in a separate view doesn't really matter because the api is already stateful (login session).